### PR TITLE
caffeine-ng: update to 4.0.2.

### PR DESCRIPTION
**Pull request by icp1994**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/caffeine-ng/template [hunk 1]
```diff
@@ -1,26 +1,19 @@
 # Template file for 'caffeine-ng'
 pkgname=caffeine-ng
-version=3.5.1
-revision=2
+version=4.0.2
+revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools_scm"
 depends="desktop-file-utils gtk+3 hicolor-icon-theme libnotify
```
Comment (classabbyamp) [inline srcpkgs/caffeine-ng/template:7]: changelog lists a bunch of deps no longer needed

File: srcpkgs/caffeine-ng/template [hunk 2]
```diff
@@ -1,26 +1,19 @@
 # Template file for 'caffeine-ng'
 pkgname=caffeine-ng
-version=3.5.1
-revision=2
+version=4.0.2
+revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools_scm"
```
Comment (classabbyamp) [inline srcpkgs/caffeine-ng/template:6]: needs scdoc for manpages

File: srcpkgs/caffeine-ng/template (hunk 1 above)
Comment (icp1994) [reply srcpkgs/caffeine-ng/template:7]: The version at the top of the changelog doesn't have a [release](https://codeberg.org/WhyNotHugo/caffeine-ng/tags) yet.

File: srcpkgs/caffeine-ng/template (hunk 2 above)
Comment (icp1994) [reply srcpkgs/caffeine-ng/template:6]: Same as above, it's still there within the [source](https://codeberg.org/WhyNotHugo/caffeine-ng/src/tag/v4.0.2/share/man/man1) in this version. The next version won't even adhere to the `build_style=python3-module` by the looks of it.

File: srcpkgs/caffeine-ng/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/caffeine-ng/template:7]: aha, i misread

**Pull request merged**